### PR TITLE
Fix spelling, mostly of "paren*"

### PR DESCRIPTION
**Pull request by StephenWall**

### CODE REVIEW

File: indent/pgsql.vim [hunk 1]
```diff
@@ -44,10 +44,22 @@ set cpo&vim
 " IS is excluded, since it is difficult to determine when the
 " ending block is (especially for procedures/functions).
 let s:SQLBlockStart = '^\s*\%('.
-                \ 'if\|else\|elseif\|elsif\|'.
-                \ 'while\|loop\|do\|for\|'.
-                \ 'declare\|begin\|'.
-                \ 'case\|when\|merge\|exception'.
+                \ 'if\>.*\<then\|'.
```
Comment (lifepillar) [inline indent/pgsql.vim]: Can you please provide an example of what this improves over the current rule?

File: indent/pgsql.vim [hunk 2]
```diff
@@ -44,10 +44,22 @@ set cpo&vim
...
+                \ 'then\|'.
+                \ 'else\|'.
+                \ 'elseif\>.*\<then\|'.
```
Comment (lifepillar) [inline indent/pgsql.vim]: Ditto.

File: indent/pgsql.vim [hunk 3]
```diff
@@ -44,10 +44,22 @@ set cpo&vim
...
+                \ 'elsif\>.*\<then\|'.
+                \ 'while\>.*\<loop\|'.
+                \ 'for\>.*\<loop\|'.
+                \ 'foreach\>.*\<loop\|'.
```
Comment (lifepillar) [inline indent/pgsql.vim]: Ditto for this four lines, too.

File: indent/pgsql.vim (hunk 1 above)
Comment (lifepillar) [reply indent/pgsql.vim]: Never mind, it's in the commit message :)

**Comment by lifepillar** [comment]
This looks good, thanks!

The indentation's file is adapted from `sqlanywhere.sql` inside Vim distribution. You may want to suggest these fixes also there.

**Comment by StephenWall** [comment]
OK, will push them a patch.  Thanks.

File: indent/pgsql.vim (hunk 1 above)
Comment (StephenWall) [reply indent/pgsql.vim]: :)

**Pull request merged**